src/layout/_template.njk

```diff
@@ -16,9 +16,6 @@
     {% set cdn_url = (pageConfig.cdn.url if pageConfig.cdn and pageConfig.cdn.url) or "https://cdn.ons.gov.uk/sdc/design-system" %}
     {% set slash = "" if cdn_url | last == "/" else "/" %}
     {% set assetsURL = cdn_url + slash + ((pageConfig.cdn.version if pageConfig.cdn and pageConfig.cdn.version) or release_version or designSystemVersion) %}
```

Comment:
`or release_version` probably isn't needed anymore (unless other services make use of this).

Reply:
release version is used by EQ, RASRM for setting the DS version in their app.